app/services/imports/lettings_logs_import_service.rb

```diff
@@ -650,5 +650,9 @@ def create_organisation_relationship!(attributes)
 
       OrganisationRelationship.find_or_create_by!(parent_organisation_id:, child_organisation_id:)
     end
+
+    def address_given?(attributes)
+      attributes["address_line1"].present? && attributes["town_or_city"].present?
+    end
```

Comment:
Does this need to look at the postcode at all?

Reply:
(I've lost track of how that's asked about on old core, so maybe not)

Reply:
Good point.

So, I mostly went off from the fact that on this page we're only validating address_line_1 and town_or_city, so existence of those 2 fields would be enough for the data on this page not to get cleared. 
My thoughts were that if we have any address data that can be imported, we prioritise that, but perhaps we only prioritise it when. the entire address is given. I'll check on the ticket. Thanks!
<img width="1596" alt="image" src="https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/assets/54268893/db4f3bae-47a4-4dc6-91d0-7f54c5291eea">

Reply:
This seems to be fine